findspam.py

```diff
@@ -397,7 +397,7 @@ def similar_answer(post):
             sanitized_answer = strip_urls_and_tags(other_answer.body)
             ratio = similar_ratio(sanitized_body, sanitized_answer)
 
-            if ratio >= SIMILAR_THRESHOLD:
+            if ratio >= 0.7:
```

Comment:
+1 to @ArtOfCode-'s suggestion - we should not be hard-coding thresholds into conditionals if we have a variable we can use instead and then just alter.